site/src/docs/components/header/index.mdx

```diff
@@ -381,13 +381,13 @@ Any ActionBarItem with a functionality that opens a dropdown, must have an indic
       onMenuClick={(e) => e.stopPropagation()}
     >
       <Header.ActionBarItem label="Ilmoitukset" icon={<IconBell />} id="action-bar-notifications">
-        <Header.ActionBarSubItem label="Ei uusi ilmoituksia." />
+        <Header.ActionBarSubItem label="Ei uusi ilmoituksia" />
```

Comment:
Hmm... shouldn't it be "Ei uusiA ilmoituksia" ? :D :D

Reply:
Jep! 😄